lib/progress_bar.rb

```diff
@@ -1,23 +1,30 @@
-
 require 'options'
 require 'highline'
 
 class ProgressBar
   Error = Class.new(StandardError)
   ArgumentError = Class.new(Error)
 
-  attr_accessor :count, :max, :meters
+  DEFAULT_METERS = [:bar, :counter, :percentage, :elapsed, :eta, :rate]
 
-  def initialize(*args)
+  attr_accessor :count, :max, :meters, :bar, :delimiters
 
+  def initialize(*args, bar: '#', delimiters: '[]')
     @count      = 0
     @max        = 100
-    @meters     = [:bar, :counter, :percentage, :elapsed, :eta, :rate]
 
     @max        = args.shift if args.first.is_a? Numeric
     raise ArgumentError, "Max must be a positive integer" unless @max >= 0
 
-    @meters     = args unless args.empty?
+    @meters     = args.empty? ? DEFAULT_METERS : args
+
+    @bar        = bar # can be an emoji which itself can be a sequence of characters
+    raise ArgumentError, 'Bar must be a valid string' unless @bar&.is_a?(String)
```

Comment:
I had to remove the character limit because lots of new emojis are multi-sequence characters. This isn't supported everywhere but in modern terminals it is. I dont see a reason to limit this functionality and its up to the developer using this Gem to make sure the character doesnt mess up